base_report_to_printer/static/src/js/qweb_action_manager.esm.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module */
+import {registry} from "@web/core/registry";
+
+async function cupsReportActionHandler(action, options, env) {
+    if (action.report_type === "qweb-pdf") {
+        const orm = env.services.orm;
+
+        const print_action = await orm.call(
+            "ir.actions.report",
+            "print_action_for_report_name",
+            [action.report_name]
+        );
+        if (print_action && print_action.action === "server") {
+            const result = await orm.call("ir.actions.report", "print_document", [
+                action.id,
+                action.context.active_ids,
+                action.data,
+            ]);
+            if (result) {
+                env.services.notification.add(env._t("Successfully sent to printer!"));
```

Comment:
@john-herholz-dt I have this error


```
UncaughtPromiseError > TypeError
Uncaught Promise > env._t is not a function
TypeError: env._t is not a function at cupsReportActionHandler 
```

Reply:
Thanks for letting me know. I can imagine there was a change in 17.0.
How did you trigger this error?

Reply:
I set up a report to send directly to the printer and when I printed the document, it failed.

I think that should be changed to `_t(...`

** `import { _t } from "@web/core/l10n/translation";`

Reply:
I don't think the issue is related to this PR. There is no code that loads { _t  }
Can you update odoo core and try again?

Reply:
That's right, it's not there now, but you have to import `_t` to be able to use it, right?

Reply:
I think you are right. Maybe it was refactored in Odoo 17.0 but I don't know about it.
I updated this PR, can you pull and recheck?